server/tserver/src/main/java/org/apache/accumulo/tserver/tablet/Tablet.java

```diff
@@ -132,6 +132,7 @@ public class Tablet extends TabletBase {
   private final AtomicLong dataSourceDeletions = new AtomicLong(0);
 
   private volatile TabletMetadata latestMetadata;
+  private long refreshCount = 0;
```

Comment:
should this be volatile?

Reply:
I had not made that volatile because it was being read an written while a lock was held, so it did not need to be volatile.  The reason I was reading it with the lock held is because it was dependent on another variable and having two volatiles that depend on each other is really tricky and something I like to avoid.  Realized I could put both variables under a single volatile to remove this tricky part and allow reading safely outside of a lock.  Made that change in bb8ae31.